packages/react-native/src/primitives/PhoneNumberField/PhoneNumberField.tsx

```diff
@@ -51,7 +51,6 @@ export default function PhoneNumberField({
       <TextField
         {...rest}
         disabled={disabled}
-        keyboardType="phone-pad"
```

Comment:
Why are we removing the `keyboardType`?

Reply:
Confirmed phone-pad has +, - characters on both platforms